identity-account/src/account/account.rs

```diff
@@ -315,6 +316,36 @@ where
     Ok(())
   }
 
+  /// If the document has a [`RevocationBitmapService`] identified by `fragment`,
+  /// revokes all credentials identified by the given `credential_indices`.
+  pub async fn revoke_credentials(&mut self, fragment: &str, credential_indices: &[u32]) -> Result<()> {
```

Comment:
Do we need an equivalent `unrevoke_credentials` function in the `Account`? Should we move the bulk of the logic to `IotaDocument` instead so it can be used from the low-level bindings too?